### PR TITLE
7 July generators minutes

### PR DESCRIPTION
**Pull request by efeyazgan**

### CODE REVIEW

File: organization/_posts/2022/2022-07-07-generators.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+yout: plain_toc
```
Comment (hegner) [inline organization/_posts/2022/2022-07-07-generators.md:1]: I believe you meant "layout"

**Review by hegner — CHANGES_REQUESTED** [review]

File: organization/_posts/2022/2022-07-07-generators.md (hunk 1 above)
Comment (efeyazgan) [reply organization/_posts/2022/2022-07-07-generators.md:1]: Right but even more: 
---
title: "HSF Generator WG Meeting #23, 7 July 2022"
layout: plain_toc
---
I am updating it now.

**Review by graeme-a-stewart — APPROVED** [review]

**Comment by efeyazgan** [comment]
The changes requested by @hegner had been done but merging is still blocked. How to resolve this?

**Comment by graeme-a-stewart** [comment]
Hi @efeyazgan - I think merging blocks until @hegner signs off on the change. However, he's away this week. In his stead I bless the chance and I can use admin powers to merge this.

**Pull request merged**